stan/math/prim/scal/prob/normal_lcdf.hpp

```diff
@@ -53,34 +55,157 @@ inline return_type_t<T_y, T_loc, T_scale> normal_lcdf(const T_y& y,
+      // (due to instability / inaccuracy in the various approximations)
+      if (scaled_diff > 2.9) {
+        // approximation derived from Abramowitx and Stegun (1964) 7.1.26
+        t = 1.0 / (1.0 + 0.3275911 * scaled_diff);
+        t2 = square(t);
+        t4 = pow(t, 4);
+        dncdf_log
+            = 1.0
+              / (SQRT_PI
+                 * (exp(x2) - 0.254829592 + 0.284496736 * t - 1.421413741 * t2
+                    + 1.453152027 * t2 * t - 1.061405429 * t4));
+      } else if (scaled_diff > 2.5) {
+        // in the trouble area where all of the standard numerical
+        // approximations are unstable - bridge the gap using Taylor
+        // expansions of the analytic function
+        // use Taylor expansion centred around x=2.7
+        t = scaled_diff - 2.7;
+        t2 = square(t);
+        t4 = pow(t, 4);
+        dncdf_log = 0.0003849882382 - 0.002079084702 * t + 0.005229340880 * t2
+                    - 0.008029540137 * t2 * t + 0.008232190507 * t4
+                    - 0.005692364250 * t4 * t + 0.002399496363 * pow(t, 6);
+      } else if (scaled_diff > 2.1) {
+        // use Taylor expansion centred around x=2.3
+        t = scaled_diff - 2.3;
+        t2 = square(t);
+        t4 = pow(t, 4);
+        dncdf_log = 0.002846135439 - 0.01310032351 * t + 0.02732189391 * t2
+                    - 0.03326906904 * t2 * t + 0.02482478940 * t4
+                    - 0.009883071924 * t4 * t - 0.0002771362254 * pow(t, 6);
+      } else if (scaled_diff > 1.5) {
+        // use Taylor expansion centred around x=1.85
+        t = scaled_diff - 1.85;
+        t2 = square(t);
+        t4 = pow(t, 4);
+        dncdf_log = 0.01849212058 - 0.06876280470 * t + 0.1099906382 * t2
+                    - 0.09274533184 * t2 * t + 0.03543327418 * t4
+                    + 0.005644855518 * t4 * t - 0.01111434424 * pow(t, 6);
+      } else if (scaled_diff > 0.8) {
+        // use Taylor expansion centred around x=1.15
+        t = scaled_diff - 1.15;
+        t2 = square(t);
+        t4 = pow(t, 4);
+        dncdf_log = 0.1585747034 - 0.3898677543 * t + 0.3515963775 * t2
+                    - 0.09748053605 * t2 * t - 0.04347986191 * t4
+                    + 0.02182506378 * t4 * t + 0.01074751427 * pow(t, 6);
+      } else if (scaled_diff > 0.1) {
+        // use Taylor expansion centred around x=0.45
+        t = scaled_diff - 0.45;
+        t2 = square(t);
+        t4 = pow(t, 4);
+        dncdf_log = 0.6245634904 - 0.9521866949 * t + 0.3986215682 * t2
+                    + 0.04700850676 * t2 * t - 0.03478651979 * t4
+                    - 0.01772675404 * t4 * t + 0.0006577254811 * pow(t, 6);
+      } else if (10.0 * log(fabs(scaled_diff))
+                 < log(std::numeric_limits<T_partials_return>::max())) {
+        // approximation derived from Abramowitx and Stegun (1964) 7.1.26
+        // use fact that erf(x)=-erf(-x)
+        // technically only true for -inf<x<0 but seems to be accurate
+        // for -inf<x<0.1
```

Comment:
If it's technically only true for -inf<x<0 then why does it work for for -inf<x<0.1? Certainly tests pass in that region. Is it just that Abramowitz and Stegun only gave the expansion for -inf<x<0? Maybe reword

Reply:
Yes that's right, I've reworded it